scipy/signal/signaltools.py

```diff
@@ -2024,6 +2074,110 @@ def filtfilt(b, a, x, axis=-1, padtype='odd', padlen=None):
     return y
 
 
+def sosfilt(sos, x, axis=-1, zi=None):
+    """
+    Filter data along one dimension using cascaded second-order sections
+
+    Filter a data sequence, `x`, using a digital IIR filter defined by
+    `sos`. This is implemented by performing `lfilter` for each
+    second-order section.  See `lfilter` for details.
+
+    Parameters
+    ----------
+    sos : array_like
+        Array of second-order filter coefficients, must have shape
+        ``(n_sections, 6)``. Each row corresponds to a second-order
+        section, with the first three columns providing the numerator
```

Comment:
This should have a more detailed example of the format or link to another function that says it? Like each row is `[b_0, b_1, b_2, a_0, a_1, a_2]`

I started writing it up here: https://github.com/scipy/scipy/pull/3954/files#diff-bc34cfe49508f8b16faacc64bb77e5f7R733